src/mocks/api/bookmarks.js

```
@@ -0,0 +1,36 @@
+ * 주의: 페이지 새로고침 시 북마크 상태가 초기 더미 데이터로 리셋됩니다.
+ */
+import { MOCK_BOOKMARKS } from "@/mocks/data";
+
+// 런타임 북마크 상태 (메모리 내 임시 저장)
+let _bookmarks = [...MOCK_BOOKMARKS];
+
+export const fetchSchoolBookmarks = async () => {
+  return { data: { school_articles: _bookmarks } };
+};
+
+export const postBookmark = async (article_type, article_id) => {
```

ESLint

[error] 'article_type' is defined but never used.

(no-unused-vars)